implementations/rust/ockam/ockam_command/tests/fixtures/user.enrollment.ticket

```diff
@@ -1 +1 @@
+7b226f6e655f74696d655f636f6465223a2266393964376533386430393736643665323733643039663034383934653030306262383733626261666538356233633462386434303435333536313564616633222c2270726f6a656374223a7b226964223a2234386465333566332d343465622d343264312d393561372d643034323966313065633164222c226e616d65223a2264656661756c74222c2273706163655f6e616d65223a22222c226163636573735f726f757465223a222f646e73616464722f6b38732d6875622d6e67696e78696e672d613631306264343233622d633135313863323965623936633463312e656c622e75732d776573742d312e616d617a6f6e6177732e636f6d2f7463702f343031352f736572766963652f617069222c227573657273223a5b5d2c2273706163655f6964223a22222c226964656e74697479223a224932396530356139383863363233336237303432643239396562343632626537626237333134343633222c22617574686f726974795f6163636573735f726f757465223a222f646e73616464722f6b38732d6875622d6e67696e78696e672d613631306264343233622d633135313863323965623936633463312e656c622e75732d776573742d312e616d617a6f6e6177732e636f6d2f7463702f343031362f736572766963652f617069222c22617574686f726974795f6964656e74697479223a2238316132303135383362613230313031303235383335613430323832303138313538323038373063346337366161393838323231383931653536323935613035353131613338323665306637353166646234353239373336353332323364623930326161303366343034316136353464376330653035316137383139376630653032383230313831353834306165636138653538393635396366316164373533336461396538363366633761653765643236323131333763343662303131366534396633383865363838343362323236313534366334316464363963636533393563343535643263353664393263306433366130353535376163353161666266303239373038666263363038222c2276657273696f6e223a6e756c6c2c2272756e6e696e67223a6e756c6c2c226f7065726174696f6e5f6964223a6e756c6c2c22757365725f726f6c6573223a5b5d7d7d
```

Comment:
Why did it change?

Reply:
Because I simplified the `EnrollmentTicket` to use the `Project` data type directly. Is this going to be a problem with the Controller maybe?

Reply:
I don't think so, but let's ask @polvorin

Reply:
Since the secure channels protocol changes also changes the ticket format, we should know with that PR first.